src/components/Contacts/ContactDetails/ContactDetailsHeader/DeleteContactModal/DeleteContactModal.tsx

```diff
@@ -19,18 +20,37 @@ import {
 import Modal from 'src/components/common/Modal/Modal';
 import { useAccountListId } from 'src/hooks/useAccountListId';
 import { isEditableSource, sourceToStr } from 'src/utils/sourceHelper';
-import { useContactSourceQuery } from './ContactSource.generated';
+import { useContactQuery } from './ContactSource.generated';
 
 const LoadingIndicator = styled(CircularProgress)(({ theme }) => ({
   margin: theme.spacing(0, 1, 0, 0),
 }));
 
+interface CreateEmailLinkProps {
+  partnerAccountNumbers: string[];
+  contactName: string;
+}
+export const createEmailLink = ({
+  partnerAccountNumbers,
+  contactName,
+}: CreateEmailLinkProps) => {
+  return `mailto:${
+    process.env.DONATION_SERVICES_EMAIL
+  }?subject=Request+contact+deletion&body=${encodeURIComponent(
+    'Dear Donation Services,\nPlease could you remove the following contact:' +
```

Comment:
```suggestion
    'Dear Donation Services,\n\nPlease could you remove the following contact:' +
```

Reply:
⬆️ @dr-bizz And this one?

Reply:
I changed it because we're getting more info than just the contactSource, we're now getting all the people on the contact